inc/switchtec/switchtec.h

```diff
@@ -399,6 +400,8 @@ void switchtec_perror(const char *str);
 int switchtec_log_to_file(struct switchtec_dev *dev,
 		enum switchtec_log_type type, int fd, FILE *log_def_file,
 		struct switchtec_log_file_info *info);
+int switchtec_log_to_ftdc_file(struct switchtec_dev *dev,
```

Comment:
It seems switchtec_log_to_file can do the same thing, is exposing this API necessary?